LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
```

Comment:
I was wondering whether we should have 2022 here, or the year when the plugin was first created (2020). I think that was the start of the copyright.

Reply:
We should keep Copyright and update it with the correct values instead, i.e. "Copyright 2022 Curity AB"